src/renderers/shared/fiber/ReactFiberCommitWork.js

```diff
@@ -390,26 +407,19 @@ module.exports = function<T, P, I, TI, C>(
         }
         if (finishedWork.effectTag & Callback) {
           if (finishedWork.callbackList) {
-            const callbackError = callCallbacks(finishedWork.callbackList, instance);
-            firstError = firstError || callbackError;
+            const { callbackList } = finishedWork;
```

Comment:
I'd rather get rid of this pattern. People have a lot of trouble reading this and has almost no value over just `const callbackList = finishedWork.callbackList;`

The exception being at the top level scope but they're intended to just be analogous to module imports. As they should be with better build tooling.